wiki/Welcome/en.md

```diff
@@ -1,148 +1,38 @@
+## Ranking process
 
-**Note:** Achievement badges can only be unlocked online based on your current account logged-in.
+_See also: [Beatmap Ranking Procedure](/wiki/Beatmap_Ranking_Procedure)._
 
-## Chat
-*Full explanation: [Chat Console][Chat_Console wikilink]*
+The ranking process of a beatmap (a difficulty that the player can play) consists of [beatmapping](/wiki/beatmapping/#getting-started) and [modding](/wiki/modding/#getting-started).
 
-Knowing the basics on how the [Chat Console][Chat_Console wikilink] works is important to those who are interested into making new friends in our friendly environment. It's also important to understand how to use it in case someone messages you (unlikely to happen unless you are well-known), telling you tips or asking for help themselves. There is a button to open up the chat on the bottom right of the screen but it's way quicker to just press F8 on your keyboard.
+Beatmapping is the process where a creator creates a beatmap. This process includes: selecting music, timing and mapping the beatmap, and testing the beatmap; and could include: adding a video, [storyboarding](/wiki/storyboarding/#getting-started), and/or [skinning](/wiki/skinning/#getting-started). All of those can be done by yourself or with other users, sometimes referred to _storyboarders_, _skinners_, and/or _guest creators_, depending on what they do.
 
-You will have a few tabs like "\#osu!", "\#announce", "\#userlog" or if you're in a conversation with someone, you'll see their name as a separate tab. Navigating through the tabs is extremely easy, just press on the tab and type your message then press "ENTER" to send the message to the opened tab. Regardless, please do not spam (continuously sending short messages into the chat) as doing so will automatically mute your account for a few minutes and your "hard-work" removed from the chat. You can press "F9" to open the user list (called Extended Chat Console) connected to the server, and you can apply filter to narrow down your search.
+Modding is the process where users review (or commonly called "mod") a creator's beatmap in the pending (or work in progress/help) stage. Modding plays a big role in quality control because this allows creators to fix issues with their beatmap.
 
-![Chat Console friendlist][Chat_Console friendlist image]
+## osu!wiki - How you can help!
 
-Those are the basics, don't be shy to ask for help in-game or in forums, also the osu!wiki is here to help understand the game more.
+Since 26. August 2016, the osu!wiki had moved to a GitHub repository. After this change, the workflow of the osu!wiki has dramatically changed. All contributions will now go through the osu-wiki GitHub repository and will be reviewed before becoming part of the osu!wiki. At this time, contributing to the osu!wiki will require more time and slightly more effort to make your changes actually happen.
```

Comment:
has moved
remove "of the osu! wiki" after "workflow" to avoid the repetition
replace "dramatically" with "drastically"

Reply:
"All contributions will now go through the osu-wiki GitHub repository and will be reviewed before becoming part of the osu!wiki."
-> "All contributions now go through the osu-wiki GitHub repository and are reviewed before being approved and merged to the wiki."

Reply:
It's probably not a good idea to add that last sentence, as it might be a bit discouraging ?

Reply:
> replace "dramatically" with "drastically"

I was trying to find the word "drastically" but kept getting spell-checked to "dramatically" 🤣 

>It's probably not a good idea to add that last sentence, as it might be a bit discouraging ?

It might be, but I thought we should at least warn them about the time and effort that they may need to put in to contribute 🤷‍♂️  ... eh remove it still? @osuShiro